actions/marketplace.js

```diff
@@ -22,11 +22,37 @@ export function fetchPlugins() {
 
             return {plugins};
         } catch (error) {
-            return {error};
+            var resultError = error;
+
+            // If the marketplace server is unreachable, try to get the local plugins only.
+            if (error.server_error_id === 'app.plugin.marketplace_plugins.app_error') {
```

Comment:
`'app.plugin.marketplace_plugins.app_error'` is not very descriptive of unreachable marketplace service. Thoughts on renaming on the server?

Reply:
0/5 — feel free to patch.

Reply:
https://github.com/mattermost/mattermost-server/pull/13388